client/goal.c

```diff
@@ -356,7 +356,8 @@ TDNFGoal(
     dwError = SolvAddFlagsToJobs(&queueJobs, nFlags);
     BAIL_ON_TDNF_ERROR(dwError);
 
-    if (nAlterType == ALTER_UPGRADEALL)
+    if (nAlterType == ALTER_UPGRADEALL ||
+        nAlterType == ALTER_UPGRADE)
```

Comment:
This change fails for the packages which includes subpackages. For example if we provide --exclude=curl we get an error like below, since it is not able to exlcude curl-libs from the list.
```
]# ./tdnf update --security --exclude=curl
1. package curl-7.61.1-2.ph3.x86_64 is disabled
Found 1 problem(s) while resolving
Error(1301) : Solv general runtime error
```

Reply:
That is was a valid error. We must skip this error where there are excludes.